README.md

```diff
@@ -286,6 +123,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
```

Comment:
The link to your `mit` license is currently broken, kindly do well to fix it as this can lead to a bad user experience and give your repo a less professional look👍🏼.

<img width="692" alt="image" src="https://github.com/mryeminthu/vet_clinic/assets/64643995/57d522d1-a150-4b35-809d-32d1f1c37707">